crates/cdk-redb/src/wallet.rs

```diff
@@ -290,6 +302,7 @@ impl WalletDatabase for RedbWalletDatabase {
         Ok(())
     }
 
+    #[instrument(skip(self))]
```

Comment:
```suggestion
    #[instrument(skip_all)]
```
Logging the full `Keys` map will be too verbose I think